cves/kernel/CVE-2016-4578.yml

```diff
@@ -55,7 +55,10 @@ description_instructions: |
 
   Your target audience is people just like you before you took any course in
   security
-description:
+description: |
```

Comment:
You could break this description into a few sentences to improve readability (maybe break the sentences between the words "initialized" and "allowed").  Additionally, you misspelled the words "access" and "information".